### PR TITLE
Improve message for WebP site health test

### LINKED ISSUE #715: Enhance the health check message when WebP is not supported
## Feature Description

Currently, if a site enables the site health check for WebP images, but the site's hosting provider does not support WebP, the only feedback we provide is the following message:

> WebP support can only be enabled by your hosting provider, so contact them for more information.

This doesn't give site owners a lot of information to help resolve the issue. Instead, it would be nice to include a link to some documentation that includes minimum requirements to support WebP that could be copy/pasted to hosting support, or could help self-hosting developers figure out why this isn't working.

In [the announcement post](https://make.wordpress.org/core/2021/06/07/wordpress-5-8-adds-webp-support/) for the inclusion of WebP images in WP 5.8, @adamsilverstein included the following:

> WebP support in the media library requires that your web server’s image processing library (WordPress supports both Imagick and LibGD) supports the WebP format. Fortunately these libraries have supported WebP for quite a while so support is widely available. If your web server does not support WebP, you will see an error message when you try to upload a WebP image.

However, this doesn't give any specifics. Is there a canonical reference somewhere that we could point to?

### PR DESCRIPTION
**Pull request by the-hercules**
Fixes #715
## Summary

Updated error message in the WordPress performance plugin regarding WebP image support. The message now informs users that WebP images require support from the web server's image processing libraries (Imagick or LibGD) and provides guidance on checking supported file formats.

### CODE REVIEW

**Comment by the-hercules** [comment]
@thelovekesh Some checks are failing due to missing labels, can you add labels to this PR?

**Review by mukeshpanchal27 — CHANGES_REQUESTED** [review]
Thanks for the PR. The link should be clickable.

Screenshot 2024-05-27 at 12 55 45 PM

**Review by joemcgill — CHANGES_REQUESTED** [review]
Prior to merging these changes, I think we need to consider whether the updated text addresses the issue. See https://github.com/WordPress/performance/issues/715#issuecomment-2139932316.

File: plugins/performance-lab/includes/site-health/webp-support/helper.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ function webp_uploads_check_webp_supported_test(): array {
 		$result['actions'] = sprintf(
 			'<p>%s</p>',
 			/* translators: Accessibility text. */
-			__( 'WebP support can only be enabled by your hosting provider, so contact them for more information.', 'performance-lab' )
+			__( 'WebP images are not currently supported by your web servers image processing libraries (Imagick or LibGD). This functionality is required to serve WebP images and improve website speed. Fortunately, these libraries have supported WebP for quite a while, so support is widely available. If your server does not support WebP, you will see an error message when you try to upload a WebP image. To check your supported file formats, go to Tools > Site Health > Info > Media Handling > Supported File Formats. For more information, visit <a href="https://learn.wordpress.org/lesson-plan/webp-images-in-wordpress/" target="_blank">WebP images in WordPress</a>.', 'performance-lab' )
```
Comment (westonruter) [inline plugins/performance-lab/includes/site-health/webp-support/helper.php]:
```suggestion
			__( 'WebP images are not currently supported by your web server's image processing libraries (Imagick or LibGD). This functionality is required to serve WebP images and improve website speed. Fortunately, these libraries have supported WebP for quite a while, so support is widely available. If your server does not support WebP, you will see an error message when you try to upload a WebP image. To check your supported file formats, go to Tools > Site Health > Info > Media Handling > Supported File Formats. For more information, visit <a href="https://learn.wordpress.org/lesson-plan/webp-images-in-wordpress/" target="_blank">WebP images in WordPress</a>.', 'performance-lab' )
```

Comment (the-hercules) [reply plugins/performance-lab/includes/site-health/webp-support/helper.php]: Resolved

**Comment by westonruter** [comment]
@the-hercules new suggested text from @adamsilverstein in https://github.com/WordPress/performance/issues/715#issuecomment-2147887299

**Comment by the-hercules** [comment]
@westonruter does this message looks good ?

WebP support can only be enabled by your hosting provider. Please contact them to inquire about switching to a plan that supports WebP, or consider switching to a host that offers this feature. For more information about WebP image support for WordPress sites, please [Learn more](https://make.wordpress.org/hosting/2022/03/30/wordpress-hosting-and-webp-support/) here.

**Comment by westonruter** [comment]
> For more information about WebP image support for WordPress sites, please [Learn more](https://make.wordpress.org/hosting/2022/03/30/wordpress-hosting-and-webp-support/) here.

I would phrase this rather as:

> [Learn more](https://make.wordpress.org/hosting/2022/03/30/wordpress-hosting-and-webp-support/) about WebP image support for WordPress sites.

I'll defer to @adamsilverstein for the ultimate wording.

**Comment by adamsilverstein** [comment]
Looks good, including suggestion from @westonruter

File: plugins/performance-lab/includes/site-health/webp-support/helper.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ function webp_uploads_check_webp_supported_test(): array {
 		$result['actions'] = sprintf(
 			'<p>%s</p>',
 			/* translators: Accessibility text. */
-			__( 'WebP images are not currently supported by your web server\'s image processing libraries (Imagick or LibGD). This functionality is required to serve WebP images and improve website speed. Fortunately, these libraries have supported WebP for quite a while, so support is widely available. If your server does not support WebP, you will see an error message when you try to upload a WebP image. To check your supported file formats, go to Tools > Site Health > Info > Media Handling > Supported File Formats. For more information, visit <a href="https://learn.wordpress.org/lesson-plan/webp-images-in-wordpress/" target="_blank">WebP images in WordPress</a>.', 'performance-lab' )
+			__( 'WebP support can only be enabled by your hosting provider. Please contact them to inquire about switching to a plan that supports WebP, or consider switching to a host that offers this feature. <a href="https://make.wordpress.org/hosting/2022/03/30/wordpress-hosting-and-webp-support/" target="_blank">Learn more</a> about WebP image support for WordPress sites.', 'performance-lab' )
```
Comment (westonruter) [inline plugins/performance-lab/includes/site-health/webp-support/helper.php]:
```suggestion
			__( 'WebP support can only be enabled by your hosting provider. Please contact them to inquire about switching to a plan that supports WebP, or consider switching to a host that offers this capability. <a href="https://make.wordpress.org/hosting/2022/03/30/wordpress-hosting-and-webp-support/" target="_blank">Learn more</a> about WebP image support for WordPress sites.', 'performance-lab' )
```

**Review by westonruter — APPROVED** [review]
LGTM, with a possible improved word choice.

**Review by westonruter — APPROVED** [review]

**Comment by the-hercules** [comment]
@mukeshpanchal27 @joemcgill Awaiting Review, I have made the suggested Changes.

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by the-hercules** [comment]
Hey @joemcgill , Could you please review this PR.

**Pull request merged**